package.json

```diff
@@ -10,7 +10,7 @@
     "extend": "^3.0.0",
     "google-auto-auth": "^0.5.2",
     "google-proto-files": "^0.9.1",
-    "grpc": "^1.1",
+    "grpc": "^1.2",
```

Comment:
You're right (I was confused by `~` versions). This means this PR is not necessary for now (current dependency spec will install 1.2.0).